<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemFormClass</class>
 <widget class="QDialog" name="AddItemFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddItemForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="9" column="5">
     <widget class="QPushButton" name="AddPowerCoef">
      <property name="text">
       <string>Add power coef</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Enter changes to unit health:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="2">
     <widget class="QLineEdit" name="changeViability">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="ToMenu">
      <property name="styleSheet">
       <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
      </property>
      <property name="text">
       <string>Go back</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="ItemNameLineEdit"/>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="AddItemButton">
      <property name="text">
       <string>Add Item</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <widget class="QLineEdit" name="ItemPower">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QListWidget" name="UnitTypes"/>
    </item>
    <item row="1" column="0" rowspan="11" colspan="2">
     <widget class="QListWidget" name="ItemsListWidget"/>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Choose type to change:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Items</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Enter changes to base power:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enter item name:&lt;b style =&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Enter power coef:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QLineEdit" name="PowerCoef">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="DeleteItemButton">
      <property name="text">
       <string>Delete Item</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
